cms/static/js/spec/views/pages/container_spec.js

```diff
@@ -591,6 +591,47 @@ define(["jquery", "underscore", "underscore.string", "js/common_helpers/ajax_hel
                         });
                     });
                 });
+
+                if (global_page_options.enable_paging) {
+                    describe("Previews", function(){
+
+                        var getButtonIcon, getButtonText;
+
+                        getButtonIcon = function(containerPage){
+                            return containerPage.$('.action-toggle-preview .preview-arrow');
+                        };
+
+                        getButtonText = function(containerPage) {
+                            return containerPage.$('.action-toggle-preview .preview-text').text().trim();
+                        };
+
+                        it('initially shows loading icon and reads "Loading..."', function(){
+                            containerPage = getContainerPage();
+                            expect(getButtonIcon(containerPage)).toHaveClass('icon-refresh');
+                            expect(getButtonText(containerPage)).toBe("${_('Loading...')}");
```

Comment:
We want to get rid of "..." from these sort of strings. It's an English idiom that doesn't translate.